kernel/kernbench.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2017 IBM
+# Author: Sudeesh John <sudeesh@linux.vnet.ibm.com>
+# Author: Harish Sriram <harish@linux.vnet.ibm.com>
+#
+# Based on code by "mbligh@google.com (Martin Bligh)"
+# https://github.com/autotest/autotest-client-tests/commits/master/kernbench
+
+import os
+import logging
+import re
+import platform
+
+from avocado import Test
+from avocado import main
+from avocado.utils import build
+from avocado.utils import process
+from avocado.utils import kernel
+from avocado.utils import cpu
+from avocado.utils import distro
+from avocado.utils.software_manager import SoftwareManager
+
+log = logging.getLogger('avocado.test')
```

Comment:
i think we can drop it as any way it's up to you

Reply:
sure